pkg/util/httprequest/httprequest_test.go

```diff
@@ -156,7 +135,7 @@ func TestSchemeHost(t *testing.T) {
 				},
 			},
 			expectedScheme: "http",
-			expectedHost:   "route-namespace.router.default.svc.cluster.local:80",
```

Comment:
restore this... this example was captured from live data and expects the explicit port to work